web/template/objection-outcome.gohtml

```diff
@@ -0,0 +1,74 @@
+{{ template "page" . }}
+
+{{ define "title" }}Objection - {{ resolutionOutcome .Resolution.Resolution }}{{ end }}
+
+{{ define "main" }}
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <a href="{{ prefix (printf "/lpa/%s" .Resolution.Uid )}}" class="govuk-back-link">Back</a>
+
+            <h1 class="govuk-heading-l app-!-embedded-hide">Objection - {{ resolutionOutcome .Resolution.Resolution }}</h1>
```

Comment:
Can drop `app-!-embedded-hide` since this won't be embedded
```suggestion
            <h1 class="govuk-heading-l">Objection - {{ resolutionOutcome .Resolution.Resolution }}</h1>
```